docs/guide/asset-management.md

```diff
@@ -79,7 +83,7 @@ const mutation = await assetLedger.transferAsset({
 ```
 
 ::: warning
-In the example above, you are transfering the asset to yourself. The receiverId is therefore your own accountId.
+In the example above, you are transfering the asset to yourself. The `receiverId` is therefore your own accountId. If you want to actually transfer the asset to someone else then input their wallet address as `receiverId`.
```

Comment:
If you want to transfer the asset to someone else, enter their wallet address as `receiverId`.